### ISSUE TITLE
blog.twitter.com URLs are mistakenly replaced

### ISSUE THREAD

**Issue by cloudrac3r** [zedeus/nitter#304]
https://nitter.snopyta.org/TwitterSafety/status/1347684877634838528

The link is replaced and leads to blog.nitter.snopyta.org - not a real domain. It should just stay as blog.twitter.com.

**Comment by austinhuang0131** [comment]
as well as help.twitter.com

**Comment by codl** [comment]
"twitter.com" seems to be replaced anywhere in a url *including in the path*, see:

https://twitter.com/codl/status/1468999428023410699
vs
https://nitter.ca/codl/status/1468999428023410699

which reveals a cool trick: if you want to know which nitter instance people are using when clicking links in your tweets, you can just add `?utm_source=twitter.com` to your links and nitter instances will diligently identify themselves 😬

**Comment by codl** [comment]
this isn't really fully fixed

https://twitter.com/codl/status/1475582438566936582:
```
test 5, a real use case

https://web.archive.org/web/2/https://twitter.com/codl
```

https://nitter.net/codl/status/1475582438566936582:
```
test 5, a real use case

https://web.archive.org/web/2/https://nitter.net/codl
```

**Comment by zedeus** [comment]
@codl fixed in 1e1e034, though this regex is getting a bit out of hand lol
![image](https://user-images.githubusercontent.com/9434921/147533387-2b9de6f5-48ce-4557-8202-2dec650ab250.png)
